src/arcade/potts/env/location/PottsLocation2D.java

```diff
@@ -64,4 +67,55 @@ Direction getSlice(Direction direction, HashMap<Direction, Integer> diameters) {
     ArrayList<Voxel> getSelected(Voxel focus, double n) {
         return Location2D.getSelected(voxels, focus, n);
     }
+
+    /**
+     * Gets the voxel at specified percentage offsets along the location's X and Y axes with the
+     * provided apicalAxis considered to be pointing up the Y axis.
+     *
+     * @param offsets the percent offsets along the location's X and Y axes
+     * @param apicalAxis the axis considered to be pointing up along the Y axis
+     * @return the voxel through which the plane of division will pass
+     */
+    public Voxel getOffsetInApicalFrame2D(ArrayList<Integer> offsets, Vector apicalAxis) {
```

Comment:
is it possible to do this as a `getOffsetInApicalFrame` and add the corresponding in PottsLocation3D? (add a not implemented error for now?) The 2D is somewhat redundant.